README.md

```diff
@@ -70,6 +70,19 @@ Options:
 
 Cleans build directory.
 
+
+#### `npm run publish`
+
+Publish extension build to the zem's registry.
```

Comment:
Publish extension, sending it for review to be listed on https://extensions.zeplin.io.